Nos/Service/CurrentUser.swift

```diff
@@ -3,15 +3,14 @@ import CoreData
 import Logger
 import Dependencies
 
-// swiftlint:disable type_body_length superfluous_disable_command
```

Comment:
???

Reply:
I think someone must have moved code out of the type, making the type_body_length superfluous, which would have caused a warning but it was ignored due to the `superfluous_disable_command`. 😵